companion/src/firmwares/opentx/opentxeeprom.cpp

```diff
@@ -1369,13 +1369,13 @@ class InputField: public TransformedField {
 
     virtual void beforeExport()
     {
-      _weight = smallGvarToEEPROM(expo.weight);
+      _weight = smallGvarImport(expo.weight);
 
-      if (IS_HORUS_OR_TARANIS(board) && version >= 216) {
-        _offset = smallGvarToEEPROM(expo.offset);
+      if ((IS_HORUS_OR_TARANIS(board) && version >= 216) || (IS_ARM(board) && version >= 218)) {
+        _offset = smallGvarImport(expo.offset);
       }
 
-      if (!IS_TARANIS(board) || version < 216) {
+      if ((!IS_ARM(board) && version >= 218) || (!IS_TARANIS(board) && version == 217) || version < 216) {
```

Comment:
Your statement would also be true for sky9x on 217. I am not sure what sky9x on 217. I am not sure if that change is right for them on 217.